src/core/public/core_app/status/components/metric_tiles.tsx

```diff
@@ -38,11 +114,19 @@ export const MetricTiles: FunctionComponent<{ metrics: Metric[] }> = ({ metrics
   </EuiFlexGrid>
 );
 
-const formatMetric = ({ value, type }: Metric) => {
+const formatMetric = ({ value, type }: Metric | MetricMeta) => {
   const metrics = Array.isArray(value) ? value : [value];
   return metrics.map((metric) => formatNumber(metric, type)).join(', ');
 };
 
 const formatMetricId = ({ name }: Metric) => {
   return name.toLowerCase().replace(/[ ]+/g, '-');
 };
+
+const formatDelayFooterTitle = (meta: MetricMeta) => {
```

Comment:
I extracted the formatting to make it easier to read. We could do something similar with the others and maybe even move them into a single formatDetails file if needed down the line.